var/spack/repos/builtin/packages/py-jsonnet/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyJsonnet(PythonPackage):
+    """A data templating language for app and tool developers based on JSON"""
+
+    homepage = "https://jsonnet.org/"
+    git = "https://github.com/google/jsonnet.git"
```

Comment:
Instead of using git commits, can we instead download tarballs from PyPI? These are generally smaller (aka faster to download), checksummable, and you can automatically detect new versions using `spack versions` or `spack checksum`.

Reply:
I'm not sure how to do that with the merge into one package....

Reply:
We can download from a GitHub URL instead, see the releases page for the project